android/src/main/java/com/swmansion/rnscreens/ScreenContainer.kt

```diff
@@ -16,9 +16,9 @@ import com.facebook.react.modules.core.ChoreographerCompat
 import com.facebook.react.modules.core.ReactChoreographer
 import com.swmansion.rnscreens.Screen.ActivityState
 
-open class ScreenContainer<T : ScreenFragment>(context: Context?) : ViewGroup(context) {
+open class ScreenContainer(context: Context?) : ViewGroup(context) {
```

Comment:
`ScreenContainer` type is no longer parameterized with `ScreenFragment` type. Instead whole fragment type variance is handled by `ScreenFragmentWrapper` interface.